devel/sdl12-compat/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup       cmake 1.0
 PortGroup       github 1.0
 
 github.setup    libsdl-org sdl12-compat 1.2.52 release-
-revision        1
+revision        2
```

Comment:
With the latest fixes just added by @jmroot, the revision is now at 2. So you'll need to bump this to 3.

Reply:
> With the latest fixes just added by @jmroot, the revision is now at 2. So you'll need to bump this to 3.

Ditto for rebasing against master, to pick up the rest of the changes.

Reply:
Thanks done.

Reply:
As I said earlier, the changes to sdl12-compat don't seem to be necessary at all.